src/services/LocalizationService.ts

```diff
@@ -31,39 +28,366 @@ class LocalizationService
+                        text2: "loremipsum97",
+                        departmentSelect: "loremipsum98"
+                    },
+                    contributors: {
+                        text1:"loremipsum99",
+                        header1: "loremipsum100",
+                        header2: "loremipsum101",
+                        dev1: "loremipsum102",
+                        dev2: "loremipsum103",
+                        dev3: "loremipsum104",
+                        githubProfile: "loremipsum105",
+                        websiteProfile: "loremipsum106",
+                        text2: "loremipsum107"
+                    },
                 },
 
-
-
-
                 en: {
-
-
-
-
+                
                     headerMenuItems: {
                         home: "Home",
                         aboutUs: "About Us",
                         rules: "Rules",
                         courses: "Courses",
-                        services: "Services",
-                        additionalGroups: "Extra groups",
-                        rapresentatives: "Rapresentatives",
-                        contributors: "Contributors"
+                        services: "All UniMi Services",
+                        additionalGroups: "Off-Topic Groups",
+                        representatives: "Presidents of Students",
+                        contributors: "Credits"
                     },
                     settingsPanel: {
                         settings: "Settings",
-                        changeTheme: "Change theme",
-                        darkTheme: "Dark Mode",
-                        lightTheme: "Light Mode",
-                        selectLanguage: "Select the language"
+                        changeTheme: "Dark Mode:",
+                        darkTheme: "On",
+                        lightTheme: "Off",
+                        selectLanguage: "Language:"
+                    },
+                    homepage: {
+                        section1: {
+                            text1: "Welcome to Network Studenti UniMi!",
+                            text2: "Our goal is to organize all the information about the University of Milan (UniMi – Università degli Studi di Milano Statale) and make them easily accessible to anyone",
+                            card1: {
+                                text1: "loremipsum108",
+                                text2: "loremipsum109"
+                            },
+                            card2: {
+                                text1: "loremipsum110",
+                                text2: "loremipsum111"
+                            }
+                        },
+                        section2: {
+                            text: "Our main links",
+                            card1: {
+                                text: "Join the Telegram group to stay updated and see all the links available!",
+                                button: "loremipsum112"
```

Comment:
The button text is "Telegram Channel"
![image](https://user-images.githubusercontent.com/52317197/119269050-0aa70900-bbf6-11eb-9bbc-9c2d31d6280b.png)